README.md

```diff
@@ -23,4 +23,4 @@ built on top of PostgreSQL, featuring:
 - built-in support for schema migrations;
 - native GraphQL support.
 
-Try the [quickstart](https://docs.geldata.com/guides/quickstart) or jump into the [docs](https://www.geldata.com/) to get started with Gel.
+Try the [quickstart](https://docs.geldata.com/learn/quickstart/overview/nextjs) or jump into the [docs](https://docs.geldata.com/) to get started with Gel.
```

Comment:
```suggestion
Try the [quickstart](https://docs.geldata.com/learn/quickstart/overview) or jump into the [docs](https://docs.geldata.com/) to get started with Gel.
```

This will automatically redirect to whatever the reader has selected as their environment, or the "first" one, which we might want to adjust.